vector/src/main/res/values/strings.xml

```diff
@@ -3283,6 +3301,14 @@
         <item quantity="one">1 active call (%1$s) · 1 paused call</item>
         <item quantity="other">1 active call (%1$s) · %2$d paused calls</item>
     </plurals>
+    <plurals name="call_active_status">
+        <item quantity="one">Active call (%1$s)</item>
+        <item quantity="other">%1$d active calls</item>
```

Comment:
I'm not sure it can work properly for every languages.
Please split into:
- 1 string when there is one active call, which takes %1$s as a parameter, as you it is now.
- 1 plural which will never be used in the code when there is only 1 active call, but will be used for other cases, without any %1$s pattern